### PR TITLE
Use the same version for spring and spring-security

### PR DESCRIPTION
**Pull request by danduk82**
this should fix https://github.com/mapfish/mapfish-print/issues/1251

### CODE REVIEW

File: ci/validate_container [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eux
```
Comment (sbrunner) [inline ci/validate_container:2]: Usually I use directly `.../bash -eux` ...

File: ci/validate_container [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+container_name=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```
Comment (sbrunner) [inline ci/validate_container:3]: You don't want to use `mapfishprint-${RANDOM}` ?

File: ci/validate_container [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+http_code=$(docker exec $container_name curl -sL -w "%{http_code}" -I localhost:8080/metrics/healthcheck -o /dev/null)
+if [[ $http_code = 501 ]] ; then
+  echo "container healthy"
+  cleanup
+  exit 0
+else
+  echo "container unhealthy"
+  cleanup
+  exit 1
+fi
```
Comment (sbrunner) [inline ci/validate_container:35]: Why not exactly the same as the first one? (and in a function?)

File: ci/validate_container [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+function cleanup(){
+  docker rm -f $container_name
+}
+
+docker run --detach  --rm --network none --name $container_name camptocamp/mapfish_print:latest
+# let the container initialize
+sleep 15
+
+http_code=$(docker exec  $container_name curl -sL -w "%{http_code}" -I localhost:8080/metrics/healthcheck -o /dev/null)
+if [[ $http_code != 501 ]] ; then
```
Comment (sbrunner) [inline ci/validate_container:14]: You don't want to change the health check?

Comment (danduk82) [reply ci/validate_container:14]: I will do another PR for that

Comment (sbrunner) [reply ci/validate_container:14]: +1 :-)

**Pull request merged**